src/xcode/ENA/ENA/Resources/Localization/tr.lproj/Localizable.legal.strings

```diff
@@ -1,4 +1,4 @@
-/* ExposureSubmissionWarnOthers */
+ExposureSubmissionQRInfo_acknowledgement_2_2/* ExposureSubmissionWarnOthers */
```

Comment:
This is not the right format and the reason why the build is not going through :/